### PR TITLE
Lab3. Iskander Bakhtiyarov

### PR DESCRIPTION
**Pull request by Feyd-Rauth**

### CODE REVIEW

**Review by sevenzing — APPROVED** [review]

File: .github/workflows/python.yml [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+  
+  snyk-python:
+    timeout-minutes: 30
+    runs-on: ubuntu-latest
+    needs: [lint, test]
+
+    steps:
+    - uses: actions/checkout@master
+    - uses: snyk/actions/setup@master
+    - uses: actions/setup-go@v1
```
Comment (Anatolay) [inline .github/workflows/python.yml]: What is "setup-go" needed for? I can only see python project here

File: .github/workflows/python.yml [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+        sarif_file: app_python/snyk.sarif
+          
+  snyk-docker:
+    timeout-minutes: 30
+    runs-on: ubuntu-latest
+    needs: [container]
+    steps:
+    - uses: actions/checkout@master
+    - uses: snyk/actions/setup@master
+    - uses: actions/setup-go@v1
```
Comment (Anatolay) [inline .github/workflows/python.yml]: Same here, why is go installed?

**Review by Anatolay — CHANGES_REQUESTED** [review]

**Comment by Anatolay** [comment]
Please, make a PR from lab3 to lab2 locally so that it is easier to see changes

File: .github/workflows/python.yml (hunk 1 above)
Comment (Feyd-Rauth) [reply .github/workflows/python.yml]: Indeed. My bad

**Review by Anatolay — APPROVED** [review]

**Review by pminina01 — APPROVED** [review]

**Review by bragov4ik — APPROVED** [review]

**Review by sevenzing — APPROVED** [review]

**Review by Ilya-Kolomin — APPROVED** [review]

**Pull request closed without merging**